util/ziputil/zip_writer.go

```diff
@@ -7,12 +7,41 @@ import (
 	"io"
 	"os"
 	"path/filepath"
+	"time"
 )
 
+type Options func(zw *zip.Writer) error
```

Comment:
I just feel like `Options` is not a good type name for a function, and it should not be plural.